caikit/core/data_model/streams/data_stream.py

```diff
@@ -257,20 +256,17 @@ def from_csv(cls, filename: str, *args, skip=0, **kwargs) -> "DataStream[List]":
         values.
 
         Args:
-            filename:  str
-                A path to a csv file that has rows corresponding to data items and columns
-                corresponding to the elements of each data item.
-            skip:  int
-                Number of lines to skip at the beginning of the csv file.  This is often useful for
-                skipping a header line.
-            args, kwargs:
-                Additional arguments passed to the `csv.reader` function.  These can be used to
-                specify the delimiter or other csv settings.
+            filename (str): A path to a csv file that has rows corresponding to
+                data items and columns corresponding to the elements of each
+                data item.
+            skip (int): Number of lines to skip at the beginning of the csv
+                file.  This is often useful for skipping a header line.
+            args, kwargs (Additional arguments passed to the `csv.reader` function.  These can be
```

Comment:
Edge case - no data type provided with colon

Reply:
Fixed. Docstring converted will now pull the description up but not put parentheses around it. However, if the first line of description is too long you may have to manually format it to be within 100 characters.